modules/aks/azure/variables.tf

```diff
@@ -75,3 +75,12 @@ variable "node_pools" {
   description = "List of node pools with minimal configuration"
   type        = map(any)
 }
+
+variable "azurerm_dns_zone" {
+  description = "This is needed as the cluster needs to know its own DNS prefixes in order to deploy the needed application"
```

Comment:
The description should describe the variable meaning, not so much why you added it.